app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,21 @@
+class FollowRequestPolicy < ApplicationPolicy
+  #create, detroy, update
+  attr_reader :user, :follow_request
+
+  def initialize(user, follow_request)
+    @user = user
+    @follow_request = follow_request
+  end
+
+  def create?
+    true
+  end
+
+  def update?
+    user == follow_request.sender || user == follow_request.recipient
+  end
+
+  def destroy?
```

Comment:
The methods that we define in our policies can inherit permissions from each other. How can we refactor this code so that the destroy method inherits permissions from the update method?